src/languages/es.ts

```diff
@@ -2284,19 +2284,18 @@ export default {
                 },
             },
             reimbursableExpenses: {
-                label: 'Gastos reembolsables de exportación como',
-                description: 'Los gastos reembolsables se exportarán como informes de gastos a Sage Intacct. Las facturas se exportarán como facturas de proveedores.',
+                label: 'Exportar gastos de bolsillo como',
+                description: 'Establezca cómo se exportan los gastos de bolsillo a Sage Intacct.',
```

Comment:
```suggestion
                description: 'Establece cómo se exportan los gastos por cuenta propia a Sage Intacct.',
```

Reply:
Gastos de bolsillo is not really a concept in Spanish.

Reply:
The other translation in the file is using `por cuenta propia`

Reply:
Ah, got it! Thank you!